src/content/api/plugins.md

```diff
@@ -99,6 +99,36 @@ compiler.hooks.myCustomHook.call(a, b, c);
 Again, see the [documentation](https://github.com/webpack/tapable) for `tapable` to learn more about the
 different hook classes and how they work.
 
+## Reporting Progress
+
+Plugins can report progress information, which is printed by [the CLI](/api/cli/) when the `--progress` argument is used. More generally, progress information is passed to [`ProgressPlugin`](/plugins/progress-plugin/), which can be used to customize how progress information is reported to the user.
```

Comment:
Suggest to remove `More generally, `. Just go with `Progress information...` for being clear.

To be honest it doesnt read well the whole paragraph.

Plugins can report progress via [`ProgressPlugin`](/plugins/progress-plugin/) which prints progress messages by [the CLI](/api/cli/). In order to enable progress reporting, pass a `--progress` argument when running webpack.

It is possible to customize the printed output by passing different arguments to the `reportProgress` function of the [`ProgressPlugin`](/plugins/progress-plugin/).

Reply:
@montogeek what do you think?

Reply:
Thanks for the feedback, and sorry for the delay!

I'm not sure about the wording of "...ProgressPlugin which prints progress messages by the CLI.", since ProgressPlugin may be used outside of the CLI.

I've updated the wording with yours, but edited this line slightly. Let me know what you think.